CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 
+## [v10.1.0] 2022-02-23
```

Comment:
Not your PR, but we should probably fix the backport note for 10.0.1.   To my knowledge it was pulled from main.